ci/test_wheel_cudf.sh

```diff
@@ -22,9 +22,22 @@ RAPIDS_PY_WHEEL_NAME="cudf_${manylinux}_${RAPIDS_PY_CUDA_SUFFIX}" rapids-downloa
 # echo to expand wildcard before adding `[extra]` requires for pip
 python -m pip install $(echo ./dist/cudf*.whl)[test]
 
+RESULTS_DIR=${RAPIDS_TESTS_DIR:-"$(mktemp -d)"}
+RAPIDS_TESTS_DIR=${RAPIDS_TESTS_DIR:-"${RESULTS_DIR}/test-results"}/
+mkdir -p "${RAPIDS_TESTS_DIR}"
+
 # Run smoke tests for aarch64 pull requests
 if [[ "$(arch)" == "aarch64" && ${RAPIDS_BUILD_TYPE} == "pull-request" ]]; then
+    rapids-logger "Run smoke tests for cudf"
     python ./ci/wheel_smoke_test_cudf.py
 else
-    python -m pytest -n 8 ./python/cudf/cudf/tests
+    rapids-logger "pytest cudf"
+    pushd python/cudf/cudf/tests
+    python -m pytest \
+      --cache-clear \
+      --junitxml="${RAPIDS_TESTS_DIR}/junit-cudf.xml" \
+      --numprocesses=8 \
```

Comment:
Did you choose to use `--numprocesses` instead of `-n` to match the conda tests? I'm fine with either option but I think (without having done any real investigation) that `-n` is more common. That may also be a dichotomy between wheels/conda (since we implemented GHA for each in parallel at roughly the same time there was less coordination/enforcement of uniformity) that we could try to unify over time.

Reply:
We've generally chosen to write the full names of flags wherever possible in our CI scripts. You're correct, this was an artifact of us doing isolated work on conda and wheels at the same time. I took this opportunity to make them match.